llama_stack/apis/common/job_types.py

```diff
@@ -10,14 +10,14 @@
 from llama_stack.schema_utils import json_schema_type
 
 
-@json_schema_type
-class Job(BaseModel):
-    job_id: str
-
-
-@json_schema_type
 class JobStatus(Enum):
     completed = "completed"
     in_progress = "in_progress"
     failed = "failed"
     scheduled = "scheduled"
+
+
+@json_schema_type
+class Job(BaseModel):
```

Comment:
can we call this `JobStatusData` or `JobStatusInfo`? I think calling it `Job` feels off

Reply:
I think we can leave that when we finalize the `Jobs` API. I'm thinking there won't be a `/status` endpoint in the future.